bugsnag-android-core/src/androidTest/java/com/bugsnag/android/AppDataOverrideTest.kt

```diff
@@ -22,13 +22,13 @@ class AppDataOverrideTest {
 
     @Before
     fun setUp() {
-        val config = Configuration("api-key")
+        val config = BugsnagTestUtils.generateConfiguration()
```

Comment:
This is a test helper method that produces a `Configuration` object - `convert` maps that to an `ImmutableConfig` object. Several of the unit tests have been updated to use this.